gemrb/core/System/VFS.cpp

```diff
@@ -465,7 +475,11 @@ bool MakeDirectory(const char* path)
 
 GEM_EXPORT char* CopyHomePath(char* outPath, ieWord maxLen)
 {
-	char* home = getenv( "HOME" );
```

Comment:
my preference would be for you to use `setenv` in some amiga specific `main` (or just ifdef the setenv there)